### PR TITLE
CIV-7977 Civils Launch Darkly Cleanup - Ticket 4

### PR DESCRIPTION
**Pull request by adam-thomas-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-7977

### Change description ###
Civils Launch Darkly Cleanup - Ticket 4

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by Gareth40343 — APPROVED** [review]

**Review by LTurkingtonHMCTS — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java [hunk 1]
```diff
@@ -118,16 +110,8 @@ void setup() {
...
 
-        @Test
-        void shouldNotUpdateSpecSupplementaryDataOnSubmittedWhenGlobalSearchOff() {
-            when(toggleService.isGlobalSearchEnabled()).thenReturn(false);
-            assignCaseToUserHandler.handle(params);
-            verify(coreCaseDataService, never()).setSupplementaryData(any(), any());
-        }
-
         @Test
         void shouldReturnSpecSupplementaryDataWhenGlobalSearchEnabled() {
```
Comment (douglasrice) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java:114]: Please rename this test by removing GlobalSearch in it's title as it is not relevant anymore
ie shouldReturnSpecSupplementaryData

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandlerTest.java [hunk 2]
```diff
@@ -1173,7 +1173,6 @@ void shouldNotCopyRespondent1OrgPolicyDetailsFor1v2SameUnregisteredSolicitorScen
 
         @Test
         void shouldAssignCaseName1v2_whenCaseIs1v2GlobalSearchEnabled() {
```
Comment (douglasrice) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandlerTest.java:1175]: Please rename test as globalSearch int name is no longer relevant
ie shouldAssignCaseName1v2_whenCaseIs1v2()

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java [hunk 3]
```diff
@@ -1428,7 +1428,6 @@ void shouldUpdateRespondentAndApplicantWithPartyNameAndPartyTypeDisplayValue_whe
         @Test
         void shouldAssignCaseName1v2_whenCaseIs1v2GlobalSearchEnabled() {
             // Given
```
Comment (douglasrice) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java:1430]: Remove GlobalSerach int name as not relevant
ie shouldAssignCaseName1v2_whenCaseIs1v2()

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java [hunk 4]
```diff
@@ -1462,7 +1460,6 @@ void shouldAssignCaseName2v1_whenCaseIs2v1GlobalSearchEnabled() {
         @Test
         void shouldAssignCaseName1v1_whenCaseIs1v1GlobalSearchEnabled() {
```
Comment (douglasrice) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java:1461]: Remove GlobalSerach int name as not relevant
ie shouldAssignCaseName1v1_whenCaseIs1v1()

**Review by douglasrice — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java [hunk 5]
```diff
@@ -90,16 +90,8 @@ void setup() {
...
 
-        @Test
-        void shouldNotUpdateSupplementaryDataOnSubmittedWhenGlobalSearchOff() {
-            when(toggleService.isGlobalSearchEnabled()).thenReturn(false);
-            assignCaseToUserHandler.handle(params);
-            verify(coreCaseDataService, never()).setSupplementaryData(any(), any());
-        }
-
         @Test
         void shouldReturnSupplementaryDataOnSubmittedWhenGlobalSearchOn() {
```
[the remark this answers is not in the document]
Comment (adam-thomas-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java]: Test name updated

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java (hunk 1 above)
Comment (adam-thomas-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseassignment/AssignCaseToUserHandlerTest.java]: Test name updated

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandlerTest.java (hunk 2 above)
Comment (adam-thomas-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandlerTest.java]: Test name updated

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java (hunk 3 above)
Comment (adam-thomas-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java:1430]: Test name updated

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java (hunk 4 above)
Comment (adam-thomas-HMCTS) [reply src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java]: Test name updated

**Review by douglasrice — DISMISSED** [review]

**Pull request merged**